prov/rxm/src/rxm_ep.c

```diff
@@ -1353,6 +1415,73 @@ rxm_ep_send_common(struct rxm_ep *rxm_ep, const struct iovec *iov, void **desc,
 	}
 }
 
+static void
+rxm_ep_conn_progress_deferred_queue(struct rxm_ep *rxm_ep,
+				    struct rxm_conn *rxm_conn)
+{
+	struct rxm_tx_entry *tx_entry;
+	ssize_t ret = 0;
+	while (!dlist_empty(&rxm_conn->deferred_tx_queue) && !ret) {
+		tx_entry = container_of(rxm_conn->deferred_tx_queue.next,
+					struct rxm_tx_entry, deferred_tx_entry);
+		switch (tx_entry->state) {
+		case RXM_LMT_ACK_SENT:	/* RNDV (LMT TX ack) */
```

Comment:
We should introduce a new state, say `RXM_LMT_ACK_DEFERRED`

Reply:
Sure, it improves readability of code. Thanks!